### PR TITLE
TTS example, InternVL2.5, Mochi-1 example

### PR DESCRIPTION
**Pull request by NilayYadav**
Added two new examples

1) TTS example using Parler TTS
2) InternVL2.5-8B example 
3) Mochi-1 example

### CODE REVIEW

File: vllm/vision_models/app.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    from vllm.entrypoints.logger import RequestLogger
+    from vllm.entrypoints.openai.serving_chat import OpenAIServingChat
+    from vllm.entrypoints.openai.serving_completion import (
+        OpenAIServingCompletion,
+    )
+    from vllm.entrypoints.openai.serving_engine import BaseModelPath
+    from vllm.usage.usage_lib import UsageContext
+
+    web_app = fastapi.FastAPI(
+        title=f"OpenAI-compatible server",
```
Comment (mernit) [inline vllm/vision_models/app.py:29]: can we use the beam vLLM wrapper instead?

https://docs.beam.cloud/v2/reference/sdk#vllm

File: tts/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+```bash
+beam deploy app.py:generate_speech
+```
+
+## API Usage
+Send a POST request with:
+```bash
+{
+    "prompt": "Your text to convert to speech",
```
Comment (mernit) [inline tts/README.md:15]: would be good to add an real output from the model here

Comment (mernit) [reply tts/README.md:15]: ideally a screenshot or whatever the asset generated from the model is

**Review by mernit — CHANGES_REQUESTED** [review]

File: tts/README.md (hunk 2 above)
Comment (NilayYadav) [reply tts/README.md:17]: Added example audio

File: vllm/vision_models/app.py (hunk 1 above)
Comment (NilayYadav) [reply vllm/vision_models/app.py:29]: @mernit The model requires two A10Gs, and the VLLM class doesn't have a gpu_count option.

Comment (NilayYadav) [reply vllm/vision_models/app.py:29]: And here (https://github.com/beam-cloud/beta9/blob/3dda0e3ca199c69f688ed5358874021ad0dd1cfc/sdk/src/beta9/abstractions/integrations/vllm.py#L46C5-L46C33), it seems like the `enable_prompt_tokens_details parameter` is not being utilized. Even though I pass this argument to the VLLMArgs class, it doesn't seem to have any effect.

 I get this error ```enable_prompt_tokens_details=args.enable_prompt_tokens_details, AttributeError: 'types.SimpleNamespace' object has no attribute 'enable_prompt_tokens_details'```

File: vllm/models.py [hunk 3]
```diff
@@ -1,19 +1,25 @@
 from beam.integrations import VLLM, VLLMArgs
+from beam import Image
 
-PHI_VISION_INSTRUCT = "microsoft/Phi-3.5-vision-instruct"
+MODEL = "OpenGVLab/InternVL2_5-8B"
```
Comment (mernit) [inline vllm/models.py:4]: you need to rename the phi variable names in inference.py

File: tts/README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    "description": "Description of the voice/style"
+}
+```
+
+### Example Request:
+
+```json
+{
+    "prompt": "On Beam run AI workloads anywhere with zero complexity. One line of Python, global GPUs, full control!!!",
```
Comment (mernit) [inline tts/README.md:26]: don't ever use multiple exclamation points

File: tts/app.py [hunk 5]
```diff
@@ -0,0 +1,60 @@
+from beam import endpoint, env, Image, Output
+
+if env.is_remote():
```
Comment (mernit) [inline tts/app.py:3]: this example should be in the `audio_and_transcription` folder

**Review by mernit — CHANGES_REQUESTED** [review]

**Pull request merged**